sstables/sstables.cc

```diff
@@ -1527,6 +1527,7 @@ size_t sstable::reclaim_memory_from_components() {
             // No need to remove it from _recognized_components as the filter is still in disk.
             _components->filter = std::make_unique<utils::filter::always_present_filter>();
             memory_reclaimed_this_iteration += filter_memory_size;
+            sstlog.info("Reclaimed bloom filter of {}", get_filename());
```

Comment:
Isn't this redundant with the log that exists in `reclaim_memory_from_sstable()`?

Reply:
I have removed it in the new version.